src/index.html

```diff
@@ -5,8 +5,360 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <div class="header__img">
+        <a href="#">
+          <img src="../src/images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu is-active"> Apple </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Samsung </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Smartphones </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__item-menu"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Gadgets </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Tablets </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Photo </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Video </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="imac"
+        >
+        <p class="card__descr">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__prodcode">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+          </div>
+
+          <p class="card__label">Reviews: 5</p>
+        </div>
+
+        <div class="card__buy">
+          <p class="card__price">Price:</p>
+          <p class="card__tag">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div class="card" data-qa="card">
```

Comment:
Consider will be better if you use the article tag for the card
```suggestion
      <article class="card" data-qa="card">
```